pkg/cmd/pipelinerun/logs_test.go

```diff
@@ -1948,10 +1948,10 @@ func TestLogs_nologs_v1beta1(t *testing.T) {
 	}
 	prlo := logOptsv1beta1(prName, ns, cs, dc, fake.Streamer([]fake.Log{}), false, false)
 	output, err := fetchLogs(prlo)
-	if err == nil {
-		t.Errorf("Unexpected output: %v", output)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
 	}
-	test.AssertOutput(t, "pipelinerun has not started yet", err.Error())
+	test.AssertOutput(t, "pipelinerun is still running: Running\n", output)
```

Comment:
just one nit: 
```suggestion
	test.AssertOutput(t, "PipelineRun is still running: Running\n", output)
```
can we address the following issue https://github.com/tektoncd/cli/issues/605 otherwise
/lgtm

Reply:
addressed